### PR TITLE
feat(ui): show ⍰ as default value in case of missing variables in mustache templates

### PR DESCRIPTION
**Pull request by nitram509**
## Motivation

while developing and testing with Zeebe, the PostgreSQL DB sometimes is out of synch and makes Simple Monitor fall flat,
during rendering the mustache template. This includes the custom /error page, which is still a white page, even with all exception handlers in place.

This PR simply places a ⍰ character as default, so rendering passes through and just the special character is placed,
where the missing variable was used. This can look like this:
Screenshot 2022-01-26 at 15 10 32
 
I know this default value violates a bit the "fail fast" principle.
But I found it more annoying to analyze broken HTML/screenshots and search for exceptions in the log.
E.g. recently a field 'elementId' was not populated ... and its used multiple times - so the hunt for the broken template did start :/
This approach seems to be more indicative that some data might be corrupt - since this is a typical replacement character. .
So in summary I consider this default/replacement value a better solution over fail fast.

Any feedback is welcome.

### CODE REVIEW

**Review by saig0 — APPROVED** [review]
@nitram509 nice :rocket: 
This is a useful change. I like it :+1:
